src/uwtools/config.py

```diff
@@ -55,6 +56,9 @@ def __init__(self, config_path):
 
         self.config_path = config_path
 
+    def __repr__(self):
+        return f'Config("{self.config_path}")'
```

Comment:
This line is a bit curious. I was definitely thinking this is where we'd see something like 

``` json.dumps(self.data))```